vizro-core/docs/pages/user-guides/data.md

```diff
@@ -314,7 +314,7 @@ For example, let us extend the [dynamic data example](#dynamic-data) above to sh
         ```
 
         1. `load_iris_data` takes a single argument, `number_of_points`, with a default value of 10.
-        2. `iris` is a pandas DataFrame created by reading from the CSV file `iris.csv`.
+        2. `iris` is a pandas DataFrame created from `px.data.iris()`.
```

Comment:
We didn't remove the code annotations here as we've done with other how-to-guides, which also makes sense to me because there are so many explanations here 😬 But it might be good if we follow a consistent approach?

Reply:
I'm waiting to see if @maxschulz-COL can resolve this to our satisfaction otherwise yes, need a consistent solution across all the examples.